localisation/00_wc_sad.csv

```diff
@@ -530,113 +480,108 @@ EVTOPTA_WCSE.12;It is a great honour, sister.;;Es ist eine gro
+EVTDESC_WCOME_nzoth.12;The legions of the N'Zoth have been sent to you by the Old God himself. Powerful naga and k'thir will support you in bringing forth the Hour of Twilight.;;Die Legionen der N'Zoth wurden Euch vom alten Gott selbst gesandt. Die m�chtigen Naga und K'thir werden Euch dabei unterst�tzen, die Stunde des Zwielichts herbeizuf�hren.;;;;;;;;;;;x
 EVTOPTAC_WCOME_nzoth.12;All hail N'Zoth.;;Heil N'Zoth.;;;;;;;;;;;x
-EVTDESC_WCOME_yoggsaron.12;The legions of Yogg-Saron have been sent to you by the Old God himself. Corrupted Watchers and N'raqi will support you in bringing forth the Hour of Twilight.;;Die Legionen von Yogg-Saron wurden Euch vom alten Gott selbst gesandt. Korrumpierte Beobachter und N'raqi werden Euch dabei unterst�tzen, die Stunde des Zwielichts herbeizuf�hren.;;;;;;;;;;;x
+EVTDESC_WCOME_yoggsaron.12;The legions of Yogg-Saron have been sent to you by the Old God himself. Corrupted watchers and n'raqi will support you in bringing forth the Hour of Twilight.;;Die Legionen von Yogg-Saron wurden Euch vom alten Gott selbst gesandt. Korrumpierte Beobachter und N'raqi werden Euch dabei unterst�tzen, die Stunde des Zwielichts herbeizuf�hren.;;;;;;;;;;;x
 EVTOPTA_WCOME_yoggsaron.12;All hail Yogg-Saron.;;Heil Yogg-Saron.;;;;;;;;;;;x
 EVTDESC_WCOME.20;A powerful general has been sent to you by C'Thun himself. He will serve you well and bring ruin upon enemies of the Old God.;;Ein m�chtiger General wurde Euch von C'Thun selbst gesandt. Er wird Euch gut dienen und Feinde des alten Gottes in den Ruin treiben.;;;;;;;;;;;x
 EVTOPTA_WCOME.20;All hail C'Thun.;;Heil C'Thun.;;;;;;;;;;;x
 EVTDESC_WCOME.21;A powerful general has been sent to you by N'Zoth himself. He will serve you well and bring ruin upon enemies of the Old God.;;Ein m�chtiger General wurde Euch von N'Zoth selbst gesandt. Er wird Euch gut dienen und Feinde des alten Gottes in den Ruin treiben.;;;;;;;;;;;x
 EVTOPTA_WCOME.21;All hail N'Zoth.;;Heil N'Zoth.;;;;;;;;;;;x
 EVTDESC_WCOME.22;A powerful general has been sent to you by Yogg-Saron himself. He will serve you well and bring ruin upon enemies of the Old God.;;Ein m�chtiger General wurde Euch von Yogg-Saron selbst gesandt. Er wird Euch gut dienen und Feinde des alten Gottes in den Ruin treiben.;;;;;;;;;;;x
 EVTOPTA_WCOME.22;All hail Yogg-Saron.;;Heil Yogg-Saron.;;;;;;;;;;;x
-EVTDESC_WCOME.25;A monstrous C'Thrax general has been sent to you by your Master himself. He will serve you well and bring ruin upon enemies of the Old God.;;Ein monstr�ser C'Thrax-General wurde von Eurem Meister selbst zu Euch geschickt. Er wird Euch gut dienen und Feinde des alten Gottes in den Ruin treiben.;;;;;;;;;;;x
+EVTDESC_WCOME.25;A monstrous C'Thraxxi general has been sent to you by your Master himself. He will serve you well and bring ruin upon enemies of the Old God.;;Ein monstr�ser C'Thraxxi-General wurde von Eurem Meister selbst zu Euch geschickt. Er wird Euch gut dienen und Feinde des alten Gottes in den Ruin treiben.;;;;;;;;;;;x
```

Comment:
C'Thraxxi is plural, so isn't it correct to say C'Thrax general?

Reply:
You're probably right there. Changing it.